src/emulate.c

```diff
@@ -978,6 +987,10 @@ static block_t *block_find_or_translate(riscv_t *rv)
     return next;
 }
 
+#if RV32_HAS(JIT)
+typedef void (*exec_tired1_block_func_t)(riscv_t *rv, uintptr_t);
```

Comment:
Replace `exec_tired1_block_func_t ` with `exec_block_func_t ` because we don't have to distinguish from tier-1 and tier-2 compilers at the moment.

Reply:
In addition, "tired1" is considered as a typo.